feature/experimental/policy/policy_vrf_selection/otg_tests/base_vrf_selection/base_vrf_selection_test.go

```diff
@@ -44,6 +44,7 @@ const (
 	ipipProtocol      = 4
 	ipv6ipProtocol    = 41
 	ipv4Address       = "198.18.0.1/32"
+	flowSrcIp         = "198.18.0.1"
```

Comment:
Can we revert this for now? We want IP packets both with and without source as "198.18.0.1". I would add a followup PR to add that logic.